terraform_files/baremetal/main.tf

```diff
@@ -162,24 +162,24 @@ resource "libvirt_domain" "worker" {
 
 data "libvirt_network_dns_host_template" "api" {
   count    = 1
-  ip       = var.bootstrap_in_place ? var.libvirt_master_ips[count.index][0] : var.api_vip
+  ip       = var.bootstrap_in_place && !var.ipv6 ? var.libvirt_master_ips[count.index][0] : var.api_vip
```

Comment:
can you explain the logic please ?

Reply:
When using ipv6, we do not know the ips before and we cannot set before.  Therefore, for DNS, we set all to point to api-vip, and when the node IP is known, we set the api-vip to point to the node-ip

Reply:
it was added to support adding dns for single node and single node have no api vip. so i don't understand the logic